libs/variant/include/variant/variant.hpp

```diff
@@ -203,6 +205,12 @@ class Variant
   ConstByteArray string_;                 ///< The string value of the variant
   VariantList    array_;                  ///< The array value of the variant
   VariantObject  object_;                 ///< The object value of the variant
+
+  template <typename T, typename D>
+  friend struct serializers::MapSerializer;
+
+  template <typename T, typename D>
+  friend struct serializers::ForwardSerializer;
```

Comment:
This is not necessary - public interface of the Variant type offers all what is needed to do (de)serialisation with zero (or virtually zero) performance impact when compiler optimisation enabled in release mode.

Reply:
done